amplify/backend/function/bridgeXYZMutation/src/handlers/updateExternalAccount.js

```diff
@@ -56,6 +41,26 @@ const updateExternalAccountHandler = async (
     account,
   );
 
+  const deletePromises = externalAccounts.map(async ({ id }) => {
+    const deleteAccountRes = await fetch(
+      `${apiUrl}/v0/customers/${bridgeCustomerId}/external_accounts/${input.id}`,
```

Comment:
And btw, is there a reason why we don't actually use this `id` in the request URL but use the `input.id`?

Reply:
Oh wow good catch that's an oversight. I wonder why this was even working.

Reply:
I guess that as there was an external account already created, the `input.id` was holding the correct external account `id` 🤔